release_dashboard/lib/widgets/merge_pr_substeps.dart

```diff
@@ -102,31 +104,37 @@ class MergePrSubstepsState extends State<MergePrSubsteps> {
     });
   }
 
+  /// Updates the error object with what the conductor throws.
+  void setError(String? errorThrown) {
+    setState(() {
+      _error = errorThrown;
+    });
+  }
+
+  /// Toggle if the widget is being loaded or not.
+  void setIsLoading(bool result) {
+    setState(() {
+      _isLoading = result;
+    });
+  }
+
   @override
   Widget build(BuildContext context) {
-    final Map<ConductorStatusEntry, Object>? releaseStatus = context.watch<StatusState>().releaseStatus;
+    final StatusState statusState = context.watch<StatusState>();
+    final Map<ConductorStatusEntry, Object>? releaseStatus = statusState.releaseStatus;
     final ConductorService conductor = context.watch<StatusState>().conductor;
-    late bool isPrRequired;
-    late String newPrLink;
-
-    // TODO(Yugue): Remove the null check, since after this issue has been resolved,
-    // it is impossible for releaseStatus to be null here, https://github.com/flutter/flutter/issues/93822.
-    if (releaseStatus == null) {
```

Comment:
No need to check if `releaseStatus` is null at this point, because it can't be anymore.